src/components/Popups/TransactionPopup.tsx

```diff
@@ -15,21 +19,36 @@ const RowNoFlex = styled(AutoRow)`
   margin-right: 16px;
 `
 
-export function TransactionPopup({ hash, success, summary }: PopupContent) {
+export function TransactionPopup({ hash, success, summary, swapProtocol }: PopupContent) {
   const { chainId } = useActiveWeb3React()
 
+  const { t } = useTranslation()
   const theme = useTheme()
 
+  const isGnosisProtocolHash = swapProtocol === SwapProtocol.COW
+
+  const explorer = isGnosisProtocolHash ? t('gnosisProtocolExplorer') : t('blockExplorer')
+
+  const explorerExternalLink = (
+    <ExternalLink
+      href={
+        isGnosisProtocolHash
```

Comment:
I think is cleaner to do it as a variable link above which returns the correct link as it is in here: https://github.com/levelkdev/swapr-dapp/pull/1009/files#diff-6bf8d2c7365c6b92df4d531ac3cef0e16595df3d8e0e34ba6eb17d1a6c7b63b7R51

Reply:
Done.